<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingWidget</class>
 <widget class="QWidget" name="ReadingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadingWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>停止读取</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>开始读取</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
